_posts/2017-08-01-integrating-brat-with-micro-services.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: Integrating BRAT annotation tool with micro-services
+author: Arpita Sharma
+published: false
+---
+
+Named Entity Recognition(NER) is one of the key tasks in information extraction. iLabs provides services for NER and Brat is used for creating annotations for NER, adding on it has a wholesome interface. So, should we provide basic annotation tool to our users?
```

Comment:
Maybe start with Innodata Labs then go to its shorten form iLabs.

Actually, I'm unsure what Innodata Labs shorten form is?